DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -240,13 +240,9 @@ function InstallPackage
         $chocoinstallparams += " $cParams"
     }
     Write-Verbose -Message "Install command: 'choco install $pName $chocoinstallparams'"
-
-    $packageInstallOuput = Invoke-Expression -Command "choco install $pName $chocoinstallparams"
+    $packageInstallOuput = Invoke-ChocoLatey "install $pName $chocoinstallparams"
     Write-Verbose -Message "Package output $packageInstallOuput "
 
-    # Clear Package Cache
-    Get-ChocoInstalledPackage 'Purge'

```

Comment:
Is there a reason this has been removed?

Reply:
the purge command can cause errors and hangs code in DSC configurations.

Reply:
There are no issues submitted around the purge command. If this is causing you issues can you raise an issue we can look at? In the meantime we need to remove it from this PR as we are not fixing that here.